nrf_modem/doc/at_commands.rst

```diff
@@ -1,9 +1,13 @@
 .. _at_commands:
 
-AT commands
-###########
+AT socket
+#########
 
-You can use the socket interface to send AT commands to the LTE modem and to receive the responses.
+.. note::
+
+   The socket-based AT interface is deprecated since version v1.3.0. Use the nrf_modem_at interface instead.
```

Comment:
What is the best place where user can see current libmodem version number. Just thinking whether it's possible that this is confused with NCS release version.

Reply:
Seems the changelog in the documentation at least has information about different nrfxlib versions so I think this is very clear.